lib/main.sh

```diff
@@ -34,6 +34,14 @@ run_main()
   local LOG_FAILED_TESTS
   local AWK_PATH="$(which "awk")"
   local OLD_IFS="${IFS}"
+  local UNAME=`uname`
+
+  # Use find instead find -E on Linux
+  if [ {UNAME} == "Linux" ]; then
+      local FIND="find"
+  elif [ {UNAME} == "Darwin" ]; then
+      local FIND= "find -E"
+  fi
```

Comment:
Else something? When neither Darwin nor Linux what's happening?
And there should not be a space after the sign "=" :-)